js/trace.ts

```diff
@@ -1,8 +1,62 @@
-import { TraceInfo, pushTraceStack, popTraceStack } from "./dispatch";
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as fbs from "gen/msg_generated";
+
+export interface TraceInfo {
+  sync: boolean; // is synchronous call
+  name: string; // name of operation
+}
+
+interface TraceListStackNode {
+  list: TraceInfo[];
+  prevStackNode: TraceListStackNode | null;
+}
+
+let currTraceListStackNode: TraceListStackNode | null = null;
+
+// Push a new list to trace stack
+export function pushTraceStack(): void {
```

Comment:
Doesn't need to be exported? Suggest renaming to `pushStack()`.

Reply:
Ooops